source/compiler/qsc/src/interpret.rs

```diff
@@ -334,7 +334,7 @@ impl Interpreter {
             .expect("package should exist in the package store")
             .package;
         for global in global::iter_package(Some(map_fir_package_to_hir(package_id)), package) {
-            if let global::Kind::Term(term) = global.kind {
+            if let global::Kind::Callable(term) = global.kind {
                 let store_item_id = fir::StoreItemId {
```

Comment:
is this still true?
```
    /// Given a package ID, returns all the global items in the package.
    /// Note this does not currently include re-exports.
```

Reply:
It is. We'd have to match `global::Kind::Export` for that